functionary/.vscode/schema.json

```diff
@@ -142,7 +142,8 @@
             },
             "required": ["name", "parameters"]
           },
-          "title": "Functions"
+          "title": "Functions",
+          "additionalProperties": false
```

Comment:
These need to be at the same level as `properties` for it to get the desired effect.  Basically anywhere you see `properties {`, just throw this above it.  It's more than the two places I originally pointed out I guess.